File: src/Presentation/dotnetcore_vuejs/src/models/authentication/token.js
function Token({
  accessToken,
  refreshToken,
  refreshTokenExpiresIn,
  tokenType
}) {
  this.accessToken = accessToken;
  this.refreshToken = refreshToken;
  this.refreshTokenExpiresIn = refreshTokenExpiresIn;
  this.tokenType = tokenType;
}

export default Token;

File: src/Presentation/Api/Dtos/AuthToken/Token.cs
namespace Api.Dtos.AuthToken;

public class Token
{
    public string AccessToken { get; set; } = null!;

    public string RefreshToken { get; set; } = null!;

    public DateTime RefreshTokenExpiresIn { get; set; }

    public string TokenType { get; set; } = null!;
}
